packages/angular/jest.config.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
   moduleNameMapper: {
     '^uuid$': '<rootDir>/../../node_modules/uuid',
     '^nanoid$': '<rootDir>/../../node_modules/nanoid',
+    axios: 'axios/dist/node/axios.cjs',
```

Comment:
Annoying, but can we add a note explaining the need for this for future context?

Reply:
I was thinking it was json, but it's js!!!! will add note